src/azul/__init__.py

```diff
@@ -190,12 +190,12 @@ def num_dss_workers(self) -> int:
 
     @property
     def external_lambda_role_assumptors(self) -> MutableMapping[str, List[str]]:
-        accounts = os.environ.get('AZUL_EXTERNAL_LAMBDA_ROLE_ASSUMPTORS')
-        # This env variable should only be set for HCA deployments
```

Comment:
I said this in my last review, but I think this comment is useful and should be preserved.